tap_kafka/serialization/protobuf.py

```diff
@@ -24,11 +24,14 @@ def __call__(self, value, ctx):
                              preserving_proto_field_name=True,
                              including_default_value_fields=True)
 
+def protoc_topic_name(topic: str) -> str:
```

Comment:
It could be nice to have a better name for the method. this name is not descriptive (what does it do exactly?)

Reply:
and also it seems it is an internal method?

Reply:
and it seems it is an internal method, isn't it?

Reply:
I made some changes and tried to give a more descriptive name. Wdyt?